renovate.json

```diff
@@ -1,10 +1,14 @@
 {
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["config:js-lib", "algolia"],
+  "baseBranches": ["chore/renovateBaseBranch"],
   "packageRules": [
     {
       "packagePatterns": ["^@types"],
       "depTypeList": ["devDependencies"],
       "rangeStrategy": "replace"
     }
-  ]
+  ],
+  "prHourlyLimit": 20,
+  "prConcurrentLimit": 50
```

Comment:
it's already 50 in the algolia template https://github.com/algolia/renovate-config-algolia/blob/master/package.json

Reply:
👁️ 👄 👁️ good catch